assets/src/components/detours/diversionPage.tsx

```diff
@@ -177,9 +177,226 @@ export const DiversionPage = ({
+                    send({
+                      type: "detour.share.activate-modal.select-reason",
+                      reason: selectedReason,
+                    })
+                  }}
+                  selectedReason={selectedReason}
+                />
+              ) : snapshot.matches({
+                  "Detour Drawing": {
+                    "Share Detour": { Activating: "Confirming" },
+                  },
+                }) ? (
+                <ActivateDetour.Confirming />
+              ) : null}
+            </ActivateDetour.Modal>
+          ) : null}
+        </DetourFinishedPanel>
+      )
+    } else if (snapshot.matches({ "Detour Drawing": "Active" })) {
+      return (
+        <ActiveDetourPanel
+          directions={extendedDirections}
+          connectionPoints={[
+            connectionPoints?.start?.name ?? "N/A",
+            connectionPoints?.end?.name ?? "N/A",
+          ]}
+          missedStops={missedStops}
+          routeName={routeName ?? "??"}
+          routeDescription={routeDescription ?? "??"}
+          routeOrigin={routeOrigin ?? "??"}
+          routeDirection={routeDirection ?? "??"}
+          onNavigateBack={onClose}
+          onOpenDeactivateModal={
+            userInTestGroup(TestGroups.DetoursPilot)
+              ? () => {
+                  send({ type: "detour.active.open-deactivate-modal" })
+                }
+              : undefined
+          }
+        >
+          {snapshot.matches({
+            "Detour Drawing": { Active: "Deactivating" },
+          }) ? (
+            <DeactivateDetourModal
+              onDeactivate={() =>
+                send({ type: "detour.active.deactivate-modal.deactivate" })
+              }
+              onCancel={() =>
+                send({ type: "detour.active.deactivate-modal.cancel" })
+              }
+              routeName={routeName || "??"}
+              routeDescription={routeDescription || "??"}
+              routeOrigin={routeOrigin || "??"}
+              routeDirection={routeDirection || "??"}
+            />
+          ) : null}
+        </ActiveDetourPanel>
+      )
+    } else if (snapshot.matches({ "Detour Drawing": "Past" })) {
+      return (
+        <PastDetourPanel
+          directions={extendedDirections}
+          connectionPoints={[
+            connectionPoints?.start?.name ?? "N/A",
+            connectionPoints?.end?.name ?? "N/A",
+          ]}
+          missedStops={missedStops}
+          routeName={routeName ?? "??"}
+          routeDescription={routeDescription ?? "??"}
+          routeOrigin={routeOrigin ?? "??"}
+          routeDirection={routeDirection ?? "??"}
+          onNavigateBack={onClose}
+        />
+      )
+    } else {
+      return <></>
+    }
+  }
+
   return (
     <>
-      <article className="l-diversion-page h-100 border-box inherit-box">
+      <article className={`l-diversion-page h-100 border-box inherit-box`}>
```

Comment:
I don't _think_ it's an issue but I don't think this change is necessary?

Reply:
Ah totally right, this is a consequence of splitting up the work -- in the following branch, I insert a classname based on isMobile, and forgot to revert the interpolation here. I'll roll back the interpolation for posterity, and then the following PR will bring it back
```suggestion
      <article className="l-diversion-page h-100 border-box inherit-box">
```